libp2p/peerstore.nim

```diff
@@ -207,11 +207,11 @@ proc identify*(
       let info = await peerStore.identify.identify(stream, stream.peerId)
 
       when defined(libp2p_agents_metrics):
-        var knownAgent = "unknown"
-        if info.agentVersion.isSome and info.agentVersion.get().len > 0:
-          let shortAgent = info.agentVersion.get().split("/")[0].safeToLowerAscii()
-          if shortAgent.isOk() and KnownLibP2PAgentsSeq.contains(shortAgent.get()):
-            knownAgent = shortAgent.get()
+        var
+          knownAgent = "unknown"
+          shortAgent = info.agentVersion.get("").split("/")[0].safeToLowerAscii().get("")
+        if shortAgent.len > 0 and KnownLibP2PAgentsSeq.contains(shortAgent):
```

Comment:
is `shortAgent.len > 0` necessary?

Reply:
Not really, was just to keep the behavior same as before, removed

Reply:
Not really, was just to keep the behavior same as before, removed